apps/src/studio/levels.js

```diff
@@ -3555,44 +3555,47 @@ var js_hoc2015_event_free_markdown = [
   '',
   '<details class="hoc2015">',
   '<summary>Example project ideas</summary>',
-  '<p>**Example 1**',
+  '<p><strong>Example 1</strong>',
```

Comment:
Markdown inside HTML isn't allowed with the new parser?

Reply:
correct. Specifically, content that should be parsed as HTML must be separated from content that should be parsed as Markdown with blank lines

See https://daringfireball.net/projects/markdown/syntax#html